temporal_navigation/SemanticType semantic_type
geometry_msgs/Point p1
geometry_msgs/Point p2